mini-rx2/src/main/java/mini/rx/StateMerger.kt

```diff
@@ -0,0 +1,89 @@
+package mini.rx
+
+import io.reactivex.Flowable
+import mini.Store
+
+/**
+ * Combine multiple store state flowables into a list of type R. If multiple stores
+ * change at the same this will emit twice, one for each store.
```

Comment:
`emit twice` would be if there were only 2 stores to be merged, but it can be more, so I would change this for:
```suggestion
 * change at the same time this will emit several times, one for each store.
```
or
```suggestion
 * change at the same time this will emit more than once, one for each store.
```

Reply:
Done! 1f876b5